src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Shared/SurveyPrompt.razor

```diff
@@ -1,4 +1,4 @@
-﻿<div class="alert alert-secondary mt-4" role="alert">
+﻿<div class="alert alert-secondary mt-4">
```

Comment:
Removing `role="alert"` here is important, and fixes one of the issues described in #32390.

* Practical reason: the attribute causes screen readers to treat this as the most important content on the page, and they read it out in preference to other things like the page title
* Conceptual reason: this content is *not* an alert. No warning sirens are going off. We just want Bootstrap's "alert-like" visual styling, which is achieved via the CSS class. I guess we (specifically, me) just unthinkingly copy-pasted the content from Bootstrap's docs in the past without recognizing how the role doesn't need to match the CSS class.

Reply:
Would it make sense to put the SurveyPrompt in an `aside` element?

Reply:
Maybe? Are you aware of any behavioural differences this would cause?

As for whether the content is semantically “aside” I don’t know for sure either. It’s probably subjective. 

If you feel confident it’s the right thing to do I’m happy to add it here. Otherwise I’m also fine with leaving it.

Reply:
> If you feel confident it’s the right thing to do

Is there a phrase that describes the opposite of that? I'm there.

Reply:
OK, since it's unclear we want to do this, and since every other time I touch anything to do with a PR it gets into a multi-hour cycle of unrelated CI failures, I'm going to merge this right now while I have the chance.